code/ship/ship.cpp

```diff
@@ -3384,6 +3392,15 @@ static void parse_ship_values(ship_info* sip, const bool is_template, const bool
 		parse_game_sound("+Autoaim Lock Snd:", &sip->autoaim_lock_snd);
 		parse_game_sound("+Autoaim Lost Snd:", &sip->autoaim_lost_snd);
 	}
+	int bank_fov_count = 0;
+	while (optional_string("+Bank Autoaim FOV:")) {
```

Comment:
This should be `+Primary Bank Autoaim FOV:` or something, to disambiguate it from secondaries. Both to be a little clearer what's it's referring to and making it a bit less awkward later if someone wants to add secondary bank autoaim.